Doppler.HelloMicroservice.Test/AuthorizationTest.cs

```diff
@@ -41,10 +41,10 @@ public async Task GET_helloAnonymous_should_not_require_token(string url, HttpSt
             // Arrange
             var client = _factory.CreateClient(new WebApplicationFactoryClientOptions());
 
-            // Act
+            //Act
             var response = await client.GetAsync(url);
```

Comment:
Please, have a look to the missing space after //

Reply:
great!